css/index.css

```diff
@@ -1 +1,265 @@
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+    font-size: 62.5%;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+.container {
+    max-width: 880px;
+    margin: 0 auto;
+    font-size: 1.6rem;
+}
+
+.container .header-main {
+	display: flex;
+	margin: 2% auto;
+}
+
+.container .header-main nav {
+	display: flex;
+	justify-content: space-around;
+	width: 80%;
+	margin: 2% auto;
+}
+
+.container .header-main nav a {
+	color: lightgray;
+	text-decoration: none;
+}
+
+.container .block1 {
+	display: flex;
+	justify-content: space-between;
+	margin: 4% auto;
+}
+
+.container .block1 div {
+	margin: 10%;
+}
+
+.container .block1 div h1 {
+	font-size: 6rem;
+	text-align: center;
+}
+
+.container .block1 div button {
+	background: white;
+	width: 120px;
+	margin: 0 25%;
+}
+
+.container .block2 {
+	border-top: 2px solid black;
+	display: flex;
+	padding: 3% 0;
+}
+
+.container .block3 {
+	display: flex;
+	padding: 2% 0;
+	border-bottom: 2px solid black;
+}
+
+.container .block3 div {
+	margin-right: 1%;
+}
+
+.container .block4 {
+	margin: 3% auto;
+}
+
+.container .block5 {
+	text-align: center;
+	margin-bottom: 2%;
+}
+
+.container .code-img {
+	display: block;
+}
+
+.container .code-img img {
+	width: 100%;
+}
+
+
+
+
+
+/* Tablet */
+@media (max-width:800px) {
+
+	.container {
+		width: 100%;
+	    padding: 0 1%;
+	 }
+
+	 .container .block1 img {
+	 	display: none;
+	 }
+
+	 .container .block1 div {
+	 	display: block;
+	 }
+
+	 .container .block1 div h1 {
+	 	text-align: center;
+	 	margin-left: 50%;
+	 }
+
+	 .container .block1 div button {
+	 	margin-left: 50%;
+	 }
+
+
+	 
+  
+}
+
+/* Mobile */
+@media (max-width: 500px) {
+	.container {
+		width: 100%;
+	 }
+
+	 .container .block1 img {
+	 	display: none;
+	 	
+	 }
+
+	 .container .header-main {
+	 	display: flex;
+	 	flex-direction: column;
+	 }
+
+	 .container .header-main nav {
+	 	display: flex;
+	 	flex-direction: column;
+	 	width: 100%;
+
+	 }
+
+	 .container .header-main img {
+	 	order: -1;
+	 	width: 40%;
+	 	height: 40%;
+	 	margin-left: 30%;
+	 }
+
+	 .container .header-main nav a {
+	 	text-align: center;
+	 	color: gray;
+	 	border-bottom: 2px solid gray;
+	 	font-size: 1.5rem;
+	 	padding: 2%;
+	 }
+
+	 .container .header-main nav a:first-child {
+	 	border-top: 2px solid gray;
+	 }
+
+
+	 .container .block1 div h1 {
+	 	text-align: center;
+	 	margin-left: 25%;
```

Comment:
Don't need margin-left: 25% because of the justify-content: center it inherit from desktop view will center it.